openldap/files/ldap.conf

```diff
@@ -21,12 +21,11 @@ URI    {{ salt['pillar.get']('openldap:uri') }}
 #TIMELIMIT	15
 #DEREF		never
 {% if not salt['pillar.get']('openldap:tls_cacertdir', False) %}
-{#- only needed on Debian? #}
 # TLS certificates (needed for GnuTLS)
 TLS_CACERT   {{ salt['pillar.get']('openldap:tls_cacert',
                     '/etc/ssl/certs/ca-certificates.crt') }}
 {%- else %}
-TLS_CACERTDIR /etc/openldap/certs
+TLS_CACERTDIR {{ salt['pillar.get']('openldap:tls_cacertdir') }}
```

Comment:
shouldn't there be a default? 
like this:
{{ salt['pillar.get']('openldap:tls_cacertdir', '/etc/openldap/certs') }}